src/main/java/com/github/coleb1911/ghost2/commands/CommandDispatcher.java

```diff
@@ -72,8 +73,15 @@ public void onMessageEvent(MessageCreateEvent event) {
         if (!checkPerms(moduleOpt.get(), context)) return;
 
         // Finally kick off command thread if all checks are passed
-        Mono<?> invokeMono = Mono.fromRunnable(() -> moduleOpt.ifPresent(m -> m.invoke(context)))
+        Mono<?> invokeMono = Mono.fromRunnable(() -> moduleOpt.ifPresent(m -> {
```

Comment:
```suggestion
        Mono<?> invokeMono = Mono.fromRunnable(() -> moduleOpt.ifPresent(m -> m.invoke(context)))
```

Reply:
We don't need to catch exceptions here. Command modules should handle them internally.

Reply:
My compiler won't let me build without adding a catch here - please advise